api.yaml

```
@@ -678,6 +678,121 @@ paths:
+      get:
+        summary: Get execution results
+        description: Get the result of an execution, together with its tests, for failed tests a list of the failed steps
+        tags:
+          - Executions
+        parameters:
+          - in: path
+            name: executionId
+            required: true
+            schema:
+              description: The ID of the execution to get details
+              example: 'EXECUTION_ID'
+              type: string
+          - in: query
+            name: page
+            schema:
+              description: the page number 
+              default: 0
+              example: 0
+              type: number
+          - in: query
+            name: limit
+            schema:
+              description: limit return test
+              default: 200
+              example: 200
+              type: number
+        responses:
+          '200':
+            description: Execution result
+            content:
+              application/json:
+                  schema:
+                    type: object
+                    properties:
+                      branch:
+                        type: string
+                        description: The branch of the tests executed
+                        example: master
+                      execution:
+                        type: string
+                        description: The name of the execution
+                        example: remote-run
+                      concurrency:
+                        type: number
+                        description: The amount of tests executed in parallel
+                        example: 1
+                      executionResult:
+                        type: string
+                        description: Wether the execution passed or failed
+                        example: Failed
+                      browser:
+                        type: string
+                        description: The name of the browser the tests were executed in
+                        example: chrome
+                      failedCount:
+                        type: number
+                        description: The amount of failed tests
+                        example: 1
+                      totalTests:
+                        type: number
+                        description: The amount of tests executed
+                        example: 1
+                      link:
+                        type: string
+                        description: A link to the execution result in Testim editor
+                      turboMode:
+                        type: boolean
+                        description: Wether the execution used Testim's Turbo Mode or not
+                      duration:
+                        type: number
+                        description: The amount of time (MS) the execution took
+                      tests:
+                        type: array
+                        description: A list of the failed tests
```

These aren't the failed tests, right?
```suggestion
                        description: A list of all included tests
```

---

fixed